torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -3078,25 +3099,49 @@ class ForceFP16Annotation : public IterVisitor {
     return annotation.force_fp16_tv_set_;
   }
 
+ public:
+  static std::unordered_set<TensorView*> getBF16AnnotatedSet(Fusion* fusion) {
```

Comment:
This function could be combined with `getFP16AnnotatedSet` with a data-type argument, since the logic looks exactly the same minus the data type.